### DISCUSSION TITLE
Git status shows "Unmerged paths" but there is no difference

### DISCUSSION THREAD

**Discussion by trallnag** [copier-org/copier#1763] [Q&A]
Whenever I run `copier update`, some of the unchanged files still show up in `git status` like this:

```txt
❯ git status
On branch main
Your branch is up to date with 'origin/main'.

Unmerged paths:
  (use "git restore --staged <file>..." to unstage)
  (use "git add <file>..." to mark resolution)
        both modified:   src/sample/__init__.py
        both modified:   tests/conftest.py

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   .copier-answers.yml
        modified:   .github/workflows/deploy-to-dev.yaml
        modified:   .pre-commit-config.yaml
        modified:   Taskfile.yaml
```

Note the `Unmerged paths`. If I check the files, I see no differences. I `git add` them and continue.

Anyone else experiencing this? I don't know why and am curious.

**Comment by pawamoy** [comment]
Yes, I experience the same thing. I implemented this in Copier, and I don't know why it happens. I didn't take the time to investigate yet since the issue is only annoying ^^

**Reply by lhupfeldt** [reply]
It is very annoying when you are updating 50 projects in an automated process and constantly run into this.

**Reply by pawamoy** [reply]
Thanks for the ping, I'm looking into it. I suppose the 3-way merge sometimes actually resolves the conflicts automatically. In that case we should check for conflict markers in the file before marking it as unmerged. Since we can't rely on Git commands to do that (edit: `git diff --check` could work), I think the best way is to grep for `<<<<<<< before updating` or `>>>>>>> after updating` lines in the file?

**Reply by pawamoy** [reply]
Opened #1813
